### DISCUSSION TITLE
I suggest adding a feature to bookmark images

### DISCUSSION THREAD

**Discussion by easygao** [civitai/civitai#416] [Ideas]
I suggest adding a feature to bookmark images. Users should be able to bookmark images they've liked by clicking a heart icon so they can easily find the models and parameters related to those images later. It would also encourage more users to engage with the platform by showing their appreciation and feedback.

**Comment by monsterquiz** [comment]
YES! I thought the same!!!!
Midjourney has something like that and is very useful.
And I would suggest TWO differents feature:
1) LIKED images 
2) BOOKMARK images (to save the USEFUL images we choose)
